### PR TITLE
print curl fails

### PR DESCRIPTION
**Pull request by makk94**

### CODE REVIEW

File: shell/probe/nocturnal.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ dos=$(uname -s)-$(uname -m)
 while :
 do  
     exe=$PRELUDE_DIR/$(uuidgen)
-    location=$(curl -sL -w %{url_effective} --create-dirs -o $exe -H "token:${PRELUDE_TOKEN}" -H "dos:${dos}" -H "dat:${dat}" $api)
+    location=$(curl -sfSL -w %{url_effective} --create-dirs -o $exe -H "token:${PRELUDE_TOKEN}" -H "dos:${dos}" -H "dat:${dat}" $api)
```
Comment (privateducky) [inline shell/probe/nocturnal.sh:13]: can you post what this will show?

Comment (makk94) [reply shell/probe/nocturnal.sh:13]: w/ a bad token:
```
mahinakaholokula@Mahinas-MacBook-Pro libraries % ./shell/probe/nocturnal.sh 
curl: (22) The requested URL returned error: 403
```

w/o the `-fS` flags, there's no output at all

Comment (mireaulf) [reply shell/probe/nocturnal.sh:13]: -S: show errors
-f: return error 22 instead of a html document

**Comment by mireaulf** [comment]
Why that change in the first place?

**Pull request merged**